BeVolt/BSP/Simulator/DataGeneration/simulate.py

```diff
@@ -104,10 +110,16 @@ def main():
         BeVolt = None
         configure()
     global stdscr
+    global CANbox
     stdscr = curses.initscr()
     curses.start_color()
     curses.noecho()
     curses.cbreak()
+    #box is for CAN messages
+    CANbox = curses.newwin(7, 21, 12, 78)
+    CANbox.immedok(True)
```

Comment:
My understanding is that `immedok` forces the window to update at any change, rendering refresh unnecessary. Probably only need to stick with one of these.